test/e2e/pivoting.go

```diff
@@ -109,14 +110,16 @@ func pivoting(ctx context.Context, inputGetter func() PivotingInput) {
 	Expect(er).ToNot(HaveOccurred(), "Cannot fetch target cluster kubeconfig")
 
 	By("Remove Ironic containers from the source cluster")
-	isIronicDeployment := true
+	ironicDeploymentType := DeploymentTypeBMO
 	if ephemeralCluster == Kind {
-		isIronicDeployment = false
+		ironicDeploymentType = DeploymentTypeLocal
+	} else if os.Getenv("USE_IRSO") == True {
```

Comment:
Use `e2e_config.GetVariable()` instead of `os.Getenv()`

Reply:
Also, consider using `switch case` for better readability.

Reply:
Is it the same thing? I see some places use both `os.GetEnv` and `GetVariable`.

Reply:
Mmm, looking at the source code, I cannot use it. GetVariable expects it to be always present, USE_IRSO is not.

Reply:
I guess I can put it in `e2e_conf.yaml`. Will it work?